package.json

```diff
@@ -88,12 +89,24 @@
     "build:prod": "webpack --config webpack.prod.js --mode production",
     "build": "webpack --config webpack.dev.js --mode development",
     "watch": "webpack --config webpack.dev.js --watch --mode development",
-    "lint": "eslint meinberlin/apps meinberlin/assets --ext .js,.jsx && stylelint 'meinberlin/assets/scss/**/*.scss' --syntax scss"
+    "lint": "eslint meinberlin/apps meinberlin/assets --ext .js,.jsx && stylelint 'meinberlin/assets/scss/**/*.scss' --syntax scss",
+    "lint-staged": "lint-staged"
   },
   "browserslist": "last 3 versions, ie >= 11",
   "husky": {
     "hooks": {
-      "pre-commit": "make lint && make check-migrations"
+      "pre-commit": "make lint-quick && make check-migrations"
     }
+  },
+  "lint-staged": {
+    "*.{js,jsx}": [
+      "eslint"
+    ],
+    "*.scss": [
+      "stylelint"
+    ],
+    "*.py": [
+      "make lint-python-files"
```

Comment:
This make-target is missing, so it doesn't work!

Reply:
oupps missed that, should I also add it to the help section in make file? it's not added in a+